ScrutinyPlot/mkscrutinydatafiles.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+datadir=/afs/cern.ch/work/c/cvuosalo/popularity/
+wkdir=/tmp/cvuosalo/popularity
```

Comment:
WorkDir should be use $USER variable instead of hardcoded value. Also you need to provide a check that such dir exists and create it if necessary.